lm_eval/tasks/mgsm/direct/mgsm_direct_bn.yaml

```diff
@@ -2,5 +2,11 @@
 dataset_name: bn
 doc_to_target: '{% if answer is not none %}{{answer[17:]}}{% else %}{{answer_number|string}}{% endif %}'
 doc_to_text: '{% if answer is not none %}{{question+"\nAnswer:"}}{% else %}{{"প্রশ্ন: "+question+"\nAnswer:"}}{% endif %}'
+generation_kwargs:
+  do_sample: false
+  until:
+  - 'প্রশ্ন:'
+  - </s>
```

Comment:
Since we're including `</s>` here, perhaps we should also include `<|endoftext|>` here and in the other tasks like GSM8k?

Reply:
I agree with it. But I think more general way would be including eos token of given tokenizer. There could be many variants of such tokens, hard to list all of them, I think.

Reply:
Good point! Will add eos token into stop sequences by default.